lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.tsx

```diff
@@ -102,21 +105,25 @@ export default function BasicLTILaunchApp() {
   /**
    * Helper to handle thrown errors from from API requests.
    *
-   * @param {Error} error - Error object from request.
-   * @param {ErrorState} state
-   * @param {boolean} [retry=true] - Can the request be retried?
-   * @param {string} [authURL] -
-   *   Authorization URL association with the API request
+   * @param error - Error object from request
+   * @param state
+   * @param [retry=true] - Can the request be retried?
+   * @param [authURL] - Authorization URL association with the API request
```

Comment:
I'm not sure what the best JSDoc approach is when one just wants to add a note about the _fourth_ parameter :).

Reply:
You can add `@param`s for just the arguments where you need to provide additional information beyond the name and type.

Reply:
To confirm: It's OK to omit params that don't need comments? So, like:

```
@param [authURL] - Authorization URL association with the API request
```

would be OK on its own?